The full unit tests started to fail after the update to CBE 63.
The investigation revealed that the change in results was
introduced by a libtiff update from version 4.0.10 to 4.1.0.
Using the full git bisect, the following changeset was found.

458c211ae2d2f9ba4a9982394594a023a3c2275c is the first bad commit
commit 458c211ae2d2f9ba4a9982394594a023a3c2275c
Author: Even Rouault <even.rouault@spatialys.com>
Date:   Fri May 10 11:08:01 2019 +0200

    Creation: use SHORT type when possible for StripByteCounts/TileByteCounts

    This follows the same logic as previous commit.

This change in libtiff library caused that md5sums changed
compared to the reference file in the testsuite. Given that
it is not a bug but the file size optimization, I updated
the checksums in the reference file. The generated images
were visually inspected and no defects were observed.

Opened an issue on the project GitHub:
https://github.com/uclouvain/openjpeg/issues/1233

The patch is not suitable for upstream since "checksum approach"
has proved to be fragile with respect to libtiff updates.

--- openjpeg-2.3.1/tests/nonregression/md5refs.txt	2020-02-07 10:30:17.622326448 +0000
+++ openjpeg-2.3.1/tests/nonregression/md5refs.txt	2020-02-07 10:29:30.230587707 +0000
@@ -180,70 +180,70 @@ dacaf60e4c430916a8c2a9ebec32e71c  issue4
 d33fb5dbddb9b9b4438eb51fa27445a3  issue495.jp2_0.pgx
 27db8c35e12a5d5eb94d403d2aae2909  issue495.jp2_1.pgx
 97da625d2f2d0b75bf891d8083ce8bfb  issue495.jp2_2.pgx
-86729c5f2b74b2dfd42cb0d8e47aef79  a1_mono_tif-1.tif
-fa9b7b896536b25a7a1d8eeeacb59141  a1_mono_tif-10.tif
-e1f194f69d1c58ce8bed62cd4f1d5b6a  a1_mono_tif-11.tif
-b0ee13aa90ca4421e09a3b7b41f410c0  a1_mono_tif-12.tif
-de53251a33356e206a793fbdbaf90db2  a1_mono_tif-13.tif
-4699894fedd3758727d8288cd7adb56c  a1_mono_tif-14.tif
-012958af4207f4a779c0cdc9e18bc9be  a1_mono_tif-15.tif
-4ad682c58e63d3223914c10a6656c8ae  a1_mono_tif-16.tif
-22c2fa09a4d7b9fade6a3cddc6c6a4dc  a1_mono_tif-2.tif
-7aea003d047d6c26475d0d257a0cfdfc  a1_mono_tif-3.tif
-996c5e67a663218be90e86bff8ecad89  a1_mono_tif-4.tif
-0489a5d490588fa603a8e59af60ef8fc  a1_mono_tif-5.tif
-7f451a5ac89915c5cdc023fd8c813a3c  a1_mono_tif-6.tif
-e164a6c0219737ee05a3d55d6e3a3104  a1_mono_tif-7.tif
-c3ebfcf478b1c4fc786748813f2b5d53  a1_mono_tif-8.tif
-67adb084f1fe234f240a1d0b2698507e  a1_mono_tif-9.tif
-31650ec40241737634179fff6ad306f8  basn4a08_tif-1.tif
-abf884080bcfbf58c044a9d86bfa5e5d  basn4a08_tif-10.tif
-b0d82c12aa2c9b3ecd96c6a5b5663a8c  basn4a08_tif-11.tif
-916d97c098d9792993cc91fee4a83f77  basn4a08_tif-12.tif
-90cabbb0f401b75dcfa64e4fbb987020  basn4a08_tif-13.tif
-57643174986481d336db6ddf04b970df  basn4a08_tif-14.tif
-3cbf7868617a0dfdbeddd5c50ea96b31  basn4a08_tif-15.tif
-fb5cf848d63c61dc485c87c9246ee9c7  basn4a08_tif-16.tif
-5d7b01d98c82ad563bb28c2d83484a2a  basn4a08_tif-2.tif
-7928de544e7d9731572a625beeb2b3ee  basn4a08_tif-3.tif
-2401cebbb1d5494fcbe0d899484c342d  basn4a08_tif-4.tif
-05ea9dfd48a6cd613bf935394988ec38  basn4a08_tif-5.tif
-6dbeb5b708bbde76e204c0887da61f6b  basn4a08_tif-6.tif
-18a59ac6036ee64e92af19b7e3cd3d64  basn4a08_tif-7.tif
-dc40cc1da6de28e7e973c8ba796ca189  basn4a08_tif-8.tif
-824b776a5aa3459b77894b5f77621311  basn4a08_tif-9.tif
-59e32c45591fd3bb44fe99381a116ba1  basn6a08_tif-1.tif
-630e6fb6deba0b3efd93b610561d607a  basn6a08_tif-10.tif
-5419fec92f0e0e5907d838dacf9712b4  basn6a08_tif-11.tif
-765555e75e59de27f7b2177d04f36bc1  basn6a08_tif-12.tif
-7d20d3e026fc1ac25be16dd44e9fb2e5  basn6a08_tif-13.tif
-62384c112d5fe40aefd0a9b0b9a4bcc6  basn6a08_tif-14.tif
-325de66e18e133f2ad865fc1faa20439  basn6a08_tif-15.tif
-d725d41557658a28ab31dff74e2467fa  basn6a08_tif-16.tif
-96d91df6b10e866ea26ebbf0b8ddc7da  basn6a08_tif-2.tif
-36690c1fe565b6efc229f5c7e486de31  basn6a08_tif-3.tif
-a324032339808d5fe85d6e354f14c183  basn6a08_tif-4.tif
-64dfb6af672ea8e890e24cfb208c6755  basn6a08_tif-5.tif
-d60864a6a5c8a49a202d98ae6f5165c7  basn6a08_tif-6.tif
-086fd12fec963995fe2e405dcef7e477  basn6a08_tif-7.tif
-c3e93f61125f82a9832d0b9440468034  basn6a08_tif-8.tif
-a9723dcc0732e74c9e8cd2bf93474a7d  basn6a08_tif-9.tif
-cfe04d15cf9d615fc36357dcb3b3956b  p0_14_tif-1.tif
-9ad87e7fddc77ac85e2e92509bee2365  p0_14_tif-10.tif
-f144e26d6d5aa24d98f0415f10751025  p0_14_tif-11.tif
-38e67f9d573e61166761d5eee0104448  p0_14_tif-12.tif
-4e804903537e4c52a7e4c15bd74eb15c  p0_14_tif-13.tif
-77486f0468694b94290d0b55361498a0  p0_14_tif-14.tif
-2aa3211823a8ee41cc5061f401dfd52d  p0_14_tif-15.tif
-51be675689949dd08b6ee1427af3eb4a  p0_14_tif-16.tif
-3e34e94bd8f7380c8d159676fee9ea57  p0_14_tif-2.tif
-f623c6b98711ba8582f2b60620c6494a  p0_14_tif-3.tif
-b6f71c941e3a5b8d2547792ccec58d54  p0_14_tif-4.tif
-5a8498d473cc57318ab9d216fdc5dcc9  p0_14_tif-5.tif
-81fcdd90917efb95aed94c6522d1c188  p0_14_tif-6.tif
-951c99efbd922d8f3feb015e9ef8e350  p0_14_tif-7.tif
-6808377b760b4ef3559ba8b14ed9b91a  p0_14_tif-8.tif
-96aa7dafa873d0ce33f84bb1ff78fa9b  p0_14_tif-9.tif
+fc19057ff2d65c24daf9c9e25e34a48a  a1_mono_tif-1.tif
+66246b6bbc83c06962f034235acb9924  a1_mono_tif-10.tif
+ce1e07bdafe83a84a5df87fce2ffde6a  a1_mono_tif-11.tif
+614f1ab59ca8473f5f8b1772b7a19d24  a1_mono_tif-12.tif
+941bdcdf9103a22f7b6f66aaca3276d1  a1_mono_tif-13.tif
+38df45296861df2b44879e8a0787d43c  a1_mono_tif-14.tif
+387575ff38bed3d177776891e1b2804e  a1_mono_tif-15.tif
+ee4838fbd88ddcb73ef26df523a3bb5d  a1_mono_tif-16.tif
+6de7e8cbd95e2c465b587f2273daf9dc  a1_mono_tif-2.tif
+e1e026d7ed26e8f3334cf25a8884abbd  a1_mono_tif-3.tif
+c120fac03d1b3756a2deb9b92ba519d4  a1_mono_tif-4.tif
+3a7b84ed9061e0b13996660fc2910e8a  a1_mono_tif-5.tif
+7ac8316261d54f22e6c847fbac01542b  a1_mono_tif-6.tif
+d714670f6746931c4e7defbfbe38b249  a1_mono_tif-7.tif
+b28f4b92be5e3481d44f50f2cd7626aa  a1_mono_tif-8.tif
+2a12dcda3e9927384e7344c4ecabdcf1  a1_mono_tif-9.tif
+4c50df5b25e006041b05e8a6fb77c95e  basn4a08_tif-1.tif
+68cc9a9bc5f95474744d06ea4efb2cf3  basn4a08_tif-10.tif
+f643c00bd0673c8f6092125e38759a35  basn4a08_tif-11.tif
+cfcefece2fb08a437876d85941cdaa27  basn4a08_tif-12.tif
+1c3850831691aa8b565e4cd0d13166f9  basn4a08_tif-13.tif
+06059e0429956946ecd3b1893ad39d18  basn4a08_tif-14.tif
+71557ba6728e6641ad289b1d142acade  basn4a08_tif-15.tif
+150c663277b43d0331112f24d47fd34e  basn4a08_tif-16.tif
+9b43011e7a19079c21d65318b4a1139b  basn4a08_tif-2.tif
+125ca7b2e45fafa4e003f5adc9f11da8  basn4a08_tif-3.tif
+9fbc1a8f4d12c8152cde3e004cebd191  basn4a08_tif-4.tif
+51c6b54e9d8b53355c3f73ad813bdeef  basn4a08_tif-5.tif
+604ac42b1a9e7a75d63e97ce40e43442  basn4a08_tif-6.tif
+360d1ce74faffa1a736d5f30c22976ed  basn4a08_tif-7.tif
+2059aaa9e54c09f36d16107870c1546a  basn4a08_tif-8.tif
+07496859507882401d66d70dcf392505  basn4a08_tif-9.tif
+98f777ca80a132d8ab820d4533daa5b6  basn6a08_tif-1.tif
+9c5873a1fa5571aad9e73d36d5a4206a  basn6a08_tif-10.tif
+c05dac7d4c19bc4b78cea426e5e52430  basn6a08_tif-11.tif
+0223432f61df2508d0195f696988ddeb  basn6a08_tif-12.tif
+ccc64d14279063ed9daf371be62077f7  basn6a08_tif-13.tif
+ace0bab2c2fbb5f92a8214600df9159f  basn6a08_tif-14.tif
+9b54bda92a09bcac9870fa02b428b7e6  basn6a08_tif-15.tif
+178665d070f54f2920521c4e1cb9d5d0  basn6a08_tif-16.tif
+c4b1e96d19429137cd8871833af2ea5a  basn6a08_tif-2.tif
+44e8b5591740289d0ca52a3e19f19c22  basn6a08_tif-3.tif
+70b4f469dd29c8e99d3f0525301286b8  basn6a08_tif-4.tif
+bf7c35a2b05eecb406aab7959431a842  basn6a08_tif-5.tif
+7d2ecb1c35869ddbafd11b4896357b81  basn6a08_tif-6.tif
+2cd6ec32a0256806f46706c0ca564d9d  basn6a08_tif-7.tif
+116b611b7a358bee2c4f2695732ec357  basn6a08_tif-8.tif
+ed9ca54d25fb5b0cd5339eedfa16cbea  basn6a08_tif-9.tif
+5f97d4bbab138f99b6b125e5a5bf96bd  p0_14_tif-1.tif
+686c7a1561d73e53c000800ec0c5fa0a  p0_14_tif-10.tif
+a8585d901cc1b7cbbda6e524ecb5db35  p0_14_tif-11.tif
+17b5497c9b8a0c68739c0beae90aa432  p0_14_tif-12.tif
+a024a04b96ccb13c81cd57a5ee6ad07a  p0_14_tif-13.tif
+aabadca3f87437a32878fef7e265b23a  p0_14_tif-14.tif
+5390a77296962268b73a793467092633  p0_14_tif-15.tif
+47dc7cc71832e5739c32794a713966c3  p0_14_tif-16.tif
+1a9247cd1fb26f5fffa870e8543f6d30  p0_14_tif-2.tif
+e4c0c9481d4032ea6b7e6a08a39e9030  p0_14_tif-3.tif
+c7d6ec9b235aaff146228875e69edbaa  p0_14_tif-4.tif
+d3b8110b2284a09cfb7d5c4ffd451aff  p0_14_tif-5.tif
+6189ee17c4a276f99302ac7e296b3daa  p0_14_tif-6.tif
+b4a3b9b63681448abb7c460702de4df9  p0_14_tif-7.tif
+04deb4e9679e7971c2cd0449fcd255b7  p0_14_tif-8.tif
+254443e438ed6a5b0631d6188cc84789  p0_14_tif-9.tif
 dd15b3d487d36a3682be0679300a4319  issue235.jp2_0.pgx
 b9cd6dc76b141fb1fec15f50c1f70e01  issue235.jp2_1.pgx
 3edef2ae197ef30b08deda1b28825399  issue235.jp2_2.pgx
@@ -287,7 +287,7 @@ fc2844a9f3c8e924e349180ba9e122dd  p0_14_
 65e41ed91283b5e2f0c79ee1c867a86c  dwt_interleave_h.gsr105.jp2_0.pgx
 8d7685f1569d446787476c0a56c93750  dwt_interleave_h.gsr105.jp2_1.pgx
 ddfff2ce2df4a9102518c92a362e6d25  dwt_interleave_h.gsr105.jp2_2.pgx
-63bf755af5a1f8a478d65079dc7c8964  issue205-tif.jp2.tif
+f9678a9e12d540f768ebebaee2af8f14  issue205-tif.jp2.tif
 b01ed87dbac424bc820b2ac590e4884e  issue236-ESYCC-CDEF.jp2_0.pgx
 2635cc00b1e18ef11adcba09e845d459  issue236-ESYCC-CDEF.jp2_1.pgx
 f9c95d0aec2f6e7b814fa1d09edcdbda  issue236-ESYCC-CDEF.jp2_2.pgx
@@ -310,11 +310,11 @@ d1bb7f93f4c0eb984b2e9c54e544b7e9  broken
 b704ad4c0cfefffd78c20a54f5541265  dwt_interleave_h.gsr105.jp2_d_1_1_33_33_0.pgx
 9d7fe43cd7a50b7bbaf712926ee11980  dwt_interleave_h.gsr105.jp2_d_1_1_33_33_1.pgx
 0960b580f991ff10f693b24aa41ad58b  dwt_interleave_h.gsr105.jp2_d_1_1_33_33_2.pgx
-fa7382fd8b2e788b28b807e200dd95b9  file1.jp2-c0.tif
-ed79b7fe443955cdefba2b039ddc846a  file1.jp2-c0_1_2.tif
-ac8f6ab3acc9c692ed7c41bd62a0e1e8  file1.jp2-c0-r1.tif
-fbfcf662b6f7549574b2885490fbcf12  file1.jp2-c0-d10_20_30_40.tif
-fa7382fd8b2e788b28b807e200dd95b9  file1.jp2-c0-t0.tif
-ac8f6ab3acc9c692ed7c41bd62a0e1e8  file1.jp2-c0-t0-r1.tif
+f771b94e7224bef9be094d4434dcc6a4  file1.jp2-c0.tif
+71c38b46dbca6cf9bd429375bc454385  file1.jp2-c0_1_2.tif
+9e6b0416d415438903def6a23d300327  file1.jp2-c0-r1.tif
+09eb5584090a28fa076172d7b7df1b1b  file1.jp2-c0-d10_20_30_40.tif
+f771b94e7224bef9be094d4434dcc6a4  file1.jp2-c0-t0.tif
+9e6b0416d415438903def6a23d300327  file1.jp2-c0-t0-r1.tif
 f31bcb01c771f829054cdb013575e86a  issue1043.png
 62bc654c830efddf1b23d6e208447dab  tnsot_zero.png
